docs/pages/distribution/runtime-versions.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Runtime Versions
+---
+
+> Custom runtime versions are not supported on the classic build system (`expo build`); these apps will always use the SDK version as the basis for determining runtime compatibility. 
+
+Over-the-air updates with `expo-updates` will only work in binaries with a [compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. By default, the runtime version is the Expo SDK version: the runtime is the set of libraries in the Expo SDK. Apps built with EAS Build will include only the dependencies that are present at the time that the app binary is built, and so the Expo SDK version will often not properly describe the runtime of the app. In these cases, you can start using `runtimeVersion` and it will take precedence over the Expo SDK version.
```

Comment:
```suggestion
Over-the-air updates with `expo-updates` are only applied to binaries with a [compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. By default, the runtime version is the Expo SDK version: the runtime is the set of libraries in the Expo SDK. If you choose to customize your runtime by adding a module with native code with [Config Plugins](../guides/config-plugins.md) or removing dependencies that are not referenced in your project with EAS Build, the Expo SDK version will no longer properly describe the runtime of the app. In these cases, you should specify a `runtimeVersion` to ensure your updates are only delivered to compatible clients.
```

Reply:
@brentvatne I couldn't find documentation for removing modules not referenced in teh project with EAS build.  Would be nice to link to it here if it exists

Reply:
* "work in" is more direct and less abstract than "applied to"
* all this applies to modules without prebuild config plugins. what matters is the js–native interface

```suggestion
Over-the-air updates with `expo-updates` work only in apps with a [compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. By default, the runtime version is the Expo SDK version: the runtime is the set of libraries in the Expo SDK. If you choose to customize your runtime by adding or removing modules with native code, the Expo SDK version will no longer properly describe the runtime of the app. In these cases, you need to specify a `runtimeVersion` to ensure your updates are delivered only to compatible clients.  This `runtimeVersion` should be updated whenever you update your project's native modules and change the JS–native interface.
```

Reply:
i didn't add that point here, i'm not sure it makes sense to include the text "or removing dependencies that are not referenced in your project with EAS Build"

Reply:
To me, "work in" seems hand wavy and imprecise, but I'm not very opinionated

I was trying to preserve the point in the existing text but happy to cut it